File: regiondeployer/randomnamegenerator.fs
module regiondeployer

open System
open System.Text

let public createrandomname(prependage:string, length: int) : string =
    let chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
    let totallength = (prependage.Length + length)
    let selected: char array = Array.zeroCreate totallength
    let random = new Random()

    for index in seq { 0 .. 1 .. length } do
        let randomNum = random.Next(chars.Length)
        let nextChar = chars[randomNum]
        Array.set selected index nextChar

    selected.ToString()

File: regiondeployer/randomnamegenerator.cs
using System;
using System.Text;

namespace regiondeployer
{
    public static class NameGenerator
    {
        public static string CreateRandomName(string prependAge, int length)
        {
            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
            int totalLength = prependAge.Length + length;
            char[] selected = new char[totalLength];
            Random random = new Random();

            for (int index = 0; index < length; index++)
            {
                int randomNum = random.Next(chars.Length);
                char nextChar = chars[randomNum];
                selected[index] = nextChar;
            }

            return new string(selected);
        }
    }
}